Wrappers.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "ElectricGuitar.h"
+#include "Computer.h"
+#include "Bus.h"
+#include "MusicStore.h"
+#include "RentalService.h"
```

Comment:
Don't include what you don't use.
None of the types in these header files are being used in this header file.
They're being used in the cpp file.
- [ ] revise